build.gradle

```diff
@@ -26,6 +29,16 @@ allprojects {
   }
 }
 
+configurations.all {
+  resolutionStrategy {
+    eachDependency { details ->
+      if (details.requested.group == 'org.jacoco') {
+        details.useVersion jacocoVersion
```

Comment:
This seems a bit hacky--why is this needed?

Reply:
This is not necessary too